Jenkinsfile

```diff
@@ -50,7 +50,7 @@ pipeline {
             }
             post {
                 always {
-                    archiveArtifacts artifacts: 'build/tests-logs*/**.log'
+                    archiveArtifacts artifacts: 'build/tests-logs*/**.log*'
```

Comment:
This is no longer going to work btw because the directory was changed in the `tests/utilities` file